frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts

```diff
@@ -150,12 +151,23 @@ export class NewMeetingComponent extends BaseComponent implements OnInit, OnDest
         }
     }
 
-    getTeamMembersOfCurrentUser(teamId?: number) {
+    subscribeToSearchTeamMembers() {
+        this.memberFilterCtrl?.valueChanges.pipe(this.untilThis, debounceTime(debounceIntervalMedium)).subscribe(() => {
+            if (this.memberFilterCtrl.getRawValue()) {
+                this.searchMembersByName();
+            } else {
+                this.teamMembers = [];
+                this.getFilteredOptions();
```

Comment:
Do we still need this?

Reply:
I think yes
when the user enters a value in input field, gets the members, and then deletes the value from input, we don't send request with empty 'searchName' to backend. But we need to clear current list of members because input filed is empty

Reply:
But if you filter an empty array, what will this filtering change?

Reply:
I removed method call and just set empty array for 'teamMembers' and 'filteredOptions'